internal/versions/versions.go

```diff
@@ -41,3 +50,25 @@ func (h *handler) ListComponentVersions(ctx context.Context, params operations.L
 			ReleaseTag: h.versions.ReleaseTag,
 		})
 }
+
+func (h *handler) ListSupportedOpenshiftVersions(ctx context.Context, params operations.ListSupportedOpenshiftVersionsParams) middleware.Responder {
+	return operations.NewListSupportedOpenshiftVersionsOK().WithPayload(h.openshiftVersions)
+}
+
+func (h *handler) GetReleaseImage(openshiftVersion string) (pullSpec string) {
+	if ri, ok := os.LookupEnv("OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE"); ok && ri != "" {
+		pullSpec = ri
+	} else {
+		pullSpec = h.openshiftVersions[openshiftVersion].ReleaseImage
```

Comment:
Don't you want to fail in case of it's not found? because you will call to an installation with non existing version.